client/client-lib/Cargo.toml

```diff
@@ -21,6 +21,7 @@ bitcoin = "0.28.1"
 bitcoin_hashes = "0.10.0"
 futures = "0.3.9"
 hex = "0.4.3"
+impl-tools = "0.4.0"
```

Comment:
We could go without using this library, and generate stuff manually, but it's small, macro only, doesn't not spill over to API, and we could review and pin.